addons/fix-costume-drag/userscript.js

```diff
@@ -0,0 +1,157 @@
+// Helper functions for patching SortableHOC taken from the folders addon by GarboMuffin
+import {
+  getSortableHOCFromElement,
+  isSortableHOC,
+  verifySortableHOC,
+  setReactInternalKey,
+  getReactInternalKey,
+  TYPE_ASSETS,
+} from "../folders/module.js";
+
+export default async function ({ addon, console }) {
+  // Settings
+  let dragScrollSetting;
+  let scrollSpeedSetting;
+
+  // indexForPositionOnList taken from https://github.com/scratchfoundation/scratch-gui/blob/develop/src/lib/drag-utils.js
+  const indexForPositionOnList = ({ x, y }, boxes, isRtl) => {
```

Comment:
From the prevoius PR if this is unmodified it might make sense to split it off into its own module to avoid accdenitly changing it.